package.json

```diff
@@ -9,6 +9,9 @@
     "start": "yarn workspace example-app start",
     "start-backend": "yarn workspace example-backend start",
     "build": "lerna run build",
+    "build:api-reports": "tsc && yarn build:api-reports:only",
+    "build:api-reports:only": "ts-node -T -P scripts/tsconfig.json scripts/api-extractor.ts",
+    "build:api-docs": "yarn build:api-reports --docs",
```

Comment:
haha, to me it's like "who the hell is it that actually gets that --docs flag three levels deep"

almost like it'd actually be helpful with some duplication between the entries. `"build:api-docs": "tsc && yarn build:api-reports:only --docs"`

Reply:
`ts-node` is also a wrapper, I think we could go for a couple more layers #puffpastry